docs/setup/quick-start.md

```diff
@@ -1,6 +1,6 @@
-There are some ways to setup Sedna:
-- If you have already an existing Kubernetes environment, you can follow [this instruction](./install.md)
-- If you have none Kubernetes environment, you can follow [the all-in-one instruction](./all-in-one.md)
-- If you want to develop Sedna, you can try [the local-up instruction](./local-up.md)
+There are some ways to setup Sedna, depends on youruse case:
```

Comment:
"setup" is a noun. Its verb form is "set up".

Reply:
OK, have fixed all